releases/release-1.20/release-notes-draft.md

```diff
@@ -1,178 +1,264 @@
+Additionally, these Kubernetes snapshot primitives act as basic building blocks that unlock the ability to develop advanced, enterprise grade, storage administration features for Kubernetes: including application or cluster level backup solutions.
+Note that snapshot support will require Kubernetes distributors to bundle the Snapshot controller, Snapshot CRDs, and validation webhook. In addition, a CSI driver supporting the snapshot functionality must also be deployed on the cluster.
+
+### Non-recursive Volume Ownership (FSGroup) graduates to Beta
+By default, the `fsgroup` setting, if specified, recursively updates permissions for every file in a volume on every mount. This can make mount, and pod startup, very slow if the volume has many files. 
+This setting enables a pod to specify a `PodFSGroupChangePolicy` that indicates that volume ownership and permissions will be changed only when permission and ownership of the root directory does not match with expected permissions on the volume.
+
+### CSIDriver policy for FSGroup graduates to Beta
+The FSGroup's CSIDriver Policy is now beta in 1.20. This allows CSIDrivers to explicitly indicate if they want Kubernetes to manage permissions and ownership for their volumes via `fsgroup`. 
+
+### Security Improvements for CSI Drivers (Alpha)
+In 1.20, we introduce a new alpha feature `CSIServiceAccountToken`. This feature allows CSI drivers to impersonate the pods that they mount the volumes for. This improves the security posture in the mounting process where the volumes are ACL’ed on the pods’ service account without handing out unnecessary permissions to the CSI drivers’ service account. This feature is especially important for secret-handling CSI drivers, such as the secrets-store-csi-driver. Since these tokens can be rotated and short-lived, this feature also provides a knob for CSI drivers to receive `NodePublishVolume` RPC calls periodically with the new token. This knob is also useful when volumes are short-lived, e.g. certificates.
+
+### Introducing Graceful Node Shutdown (Alpha)
+The `GracefulNodeShutdown` feature is now in Alpha. This allows kubelet to be aware of node system shutdowns, enabling graceful termination of pods during a system shutdown. This feature can be [enabled through feature gate](https://docs.k8s.io/concepts/architecture/nodes/#graceful-node-shutdown).
+
+### Runtime log sanitation
+Logs can now be configured to use runtime protection from leaking sensitive data. [Details for this experimental feature is available in documentation](https://docs.k8s.io/concepts/cluster-administration/system-logs/#log-sanitization).
+
+### Pod resource metrics
+On-demand metrics calculation is now available through `/metrics/resources`. [When enabled](
+https://docs.k8s.io/concepts/cluster-administration/system-metrics#kube-scheduler-metrics), the endpoint will report the requested resources and the desired limits of all running pods.
```

Comment:
SIG Instrumentation notes LGTM.

Since doc changes aren't live yet, links for reference:

- https://deploy-preview-24845--kubernetes-io-vnext-staging.netlify.app/docs/concepts/cluster-administration/system-logs/#log-sanitization
- https://deploy-preview-24910--kubernetes-io-vnext-staging.netlify.app/docs/concepts/cluster-administration/system-metrics/#kube-scheduler-metrics